test/sanitize-html.test.js

```diff
@@ -6,3 +6,57 @@ describe('Sanitize HTML', () => {
     expect(data).toBe('<img src="x" />');
   });
 });
+
+describe('Sanitize HTML', () => {
+  test('<a><img> should work, but inline js should not', () => {
+    const data = sanitizeHTML(
+      '<a href="https://www.telescope.com"><img border="0" alt="W3Schools" src="logo.gif" width="100" height="100"></a>'
+    );
+    expect(data).toBe('<a href="https://www.telescope.com"><img src="logo.gif" /></a>');
+  });
+});
+
+describe('Sanitize HTML', () => {
```

Comment:
just do 1 `describe('Sanitize HTML', () => {` wrapped around all these tests, not one per test.